### PR TITLE
Remove empty bid as method of bidding strategy

### PR DESCRIPTION
**Pull request by nick-harder**
Removes the clearing of empty bids from unit operator into the bidding strategies themselves. This allows for the user to control this behavior and keep bids with volume of 0 if needed (such as for redispatch market)

### CODE REVIEW

File: assume/strategies/naive_strategies.py [hunk 1]
```diff
@@ -73,6 +73,10 @@ def calculate_bids(
             else:
                 op_time = min(op_time, 0) - 1
 
+        bids = self.remove_empty_bids(bids)
+
+        bids = self.remove_empty_bids(bids)
```
Comment (maurerle) [inline assume/strategies/naive_strategies.py:78]: why is this needed twice?

**Review by maurerle — APPROVED** [review]
If you are sure about the results of the flexable strategies tests, this is fine.

You could add two small tests which test the new functions independently to check that:
1. does not remove volume !=0
2. removes volume == 0

File: assume/strategies/naive_strategies.py (hunk 1 above)
Comment (nick-harder) [reply assume/strategies/naive_strategies.py:78]: oh yeah thats a mistake
